src/sentry/tasks/store.py

```diff
@@ -189,6 +189,23 @@ def preprocess_event_from_reprocessing(
     )
 
 
+def increment_low_priority_metrics_counter(cluster, project_id, timestamp, ttl):
```

Comment:
I think this method should not mention "low_priority" because we will collect metrics for normal queue tasks as well.

Reply:
`increment_project_event_counter`?